svelte/src/routes/packages/[slug]/+page.svelte

```diff
@@ -41,6 +48,18 @@
       props: {
         bottles
       }
+    },
+    {
+      label: "cli",
+      component: Terminal,
+      props: { project: pkg?.full_name },
+      hidden: !pty
+    },
+    {
```

Comment:
it might be better to add this only when iframe url is ready or we add a loader in the component.

Reply:
check out the way it works now.